gen_workstation.sh

```
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+container=$1
```

Shellcheck (0.10.0)

[warning] container appears unused. Verify use (or export if used externally).

(SC2034)